cmd/rook/ceph/admission.go

```diff
@@ -18,16 +18,30 @@ package ceph
 
 import (
 	"github.com/rook/rook/cmd/rook/rook"
-	"github.com/rook/rook/pkg/daemon/admission"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	operator "github.com/rook/rook/pkg/operator/ceph"
 	"github.com/spf13/cobra"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
 var (
 	admissionCmd = &cobra.Command{
 		Use:   "admission-controller",
 		Short: "Starts admission controller",
 	}
+	scheme          = runtime.NewScheme()
+	pathResourceMap = map[string]webhook.Validator{
+		"/validate-cephclusters":   &cephv1.CephCluster{},
```

Comment:
Let's define this pathResourceMap inside the `pkg/operator/ceph/server.go` file. I don't see a reason to have this in the cmd package.